conf/test_local.config

```diff
@@ -36,40 +36,39 @@ params {
 
     // fasta_pathogen  = "https://github.com/nf-core/test-datasets/raw/dualrnaseq/references/SL1344_sub.fasta"
```

Comment:
You may be aware, there's an official test repo with a dualrnaseq branch. This does need to be linked up and work.
I had some issues getting the test files working for some weird reason - which is why those lines are commented out and local files are used.
Perhaps we can tidy up these profiles and have a local as you have done, and one that uses the 'online' test data